scripts/populate_tox/config.py

```diff
@@ -236,6 +236,12 @@
         "package": "loguru",
         "num_versions": 2,
     },
+    "mcp": {
+        "package": "mcp",
+        "deps": {
+            "*": ["pytest-asyncio", "mcp>=1.15.0"],
```

Comment:
```suggestion
            "*": ["pytest-asyncio"],
```
The script will take the minimum version from `integrations/__init__.py` automatically